<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-QP0HI4H" failures="0" tests="4" name="Scripts.TestCase1" time="421.780" errors="1" timestamp="2023-05-09T21:25:36 IST" skipped="0">
  <testcase classname="Scripts.TestCase1" name="tc1" time="137.529"/>
  <system-out/>
  <testcase classname="Scripts.TestCase1" name="tc1" time="125.157"/>
  <system-out/>
  <testcase classname="Scripts.TestCase1" name="tc1" time="159.093"/>
  <system-out/>
  <testcase classname="Scripts.TestCase1" name="aftesuit" time="0.001">
    <error message="Cannot invoke &quot;com.aventstack.extentreports.reporter.ExtentHtmlReporter.flush(com.aventstack.extentreports.ReportAggregates)&quot; because &quot;this.htmlreports&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.reporter.ExtentHtmlReporter.flush(com.aventstack.extentreports.ReportAggregates)" because "this.htmlreports" is null
at Genriclib.Baseclass.aftesuit(Baseclass.java:83)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:344)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- aftesuit -->
  <system-out/>
</testsuite> <!-- Scripts.TestCase1 -->
